README.md

```diff
@@ -38,6 +38,7 @@ Of course you can judge the not recommended abbreviations. For example why don't
 * **alloc** allocate
 * **alt** alternative/alternate
 * **arg** argument
+* **arr** array
```

Comment:
I feel it's not shorter enough and I'd rather say it's a not recommended abbreviation. 
Although these things are personal. 

What do you think?

Reply:
Hi,

Indeed, `arr` might be a kind of personal abbreviation although it is used in my codes frequently.  
Or I could revert the commit, and push again after removing `arr`.   
Thanks for your reply.  

Cheers,  
Steven